### DISCUSSION TITLE
How to update linux snap installation ?

### DISCUSSION THREAD

**Discussion by ajkis** [sct/overseerr#2698] [General]
So with snap update being disabled, how can i update it ?

p.s. sudo snap refresh - output: All snaps up to date

**Comment by DCMarine** [comment]
This is a known issue as the Snap upstream has had a problem since 1.28.0

**Reply by ajkis** [reply]
Any suggestion eg unistall / new install / import new data.
What can be done ?

**Comment by stanthewizzard** [comment]
same issue same question :)
Thanks

**Reply by ajkis** [reply]
So basically there is no way to updated it ?

**Comment by slconrad** [comment]
Any updates or recommendations here?

**Comment by bkloosterboer** [comment]
Running into this one as well.  1.28.0 seems to work well enough but not being able to update the app is a pretty significant issue.
